packages/client/hmi-client/src/components/model/petrinet/model-configurations/tera-stratified-matrix.vue

```diff
@@ -109,16 +109,17 @@ const props = defineProps<{
 	id: string;
 	stratifiedMatrixType: StratifiedMatrix;
 	shouldEval: boolean;
+	matrixType?: string;
```

Comment:
Will this break if I use matrixType = "example"? 
If it has to be one of the types in `matrixTypes` would it be worth throwing into an enum?

Reply:
Not worth it right now - if this copy/paste idea moves further we shall revisit.